partcad/tests/unit/test_git_clone_retries.py

```
@@ -0,0 +1,119 @@
+from git import Repo
```

Ruff (0.8.2)

`git.Repo` imported but unused

Remove unused import: `git.Repo`

(F401)